ci/cloudbuild/dockerfiles/checkers.Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN dnf makecache && \
         python-pip \
         ShellCheck
 
-RUN cargo install typos-cli --version 1.3.9 --root /usr/local
+RUN cargo install typos-cli --version 1.16.1 --root /usr/local
```

Comment:
FWIW, the old version also appears in `doc/contributor/howto-guide-setup-development-workstation.md`.

Reply:
Ack, Thanks. I'll update that doc in a follow up PR, just so this PR can get merged as quickly as possible. :shrug:

Reply:
Fixed.